### ISSUE TITLE
No way to see all the data!

### ISSUE THREAD

**Issue by theduncanclark** [kiln/template-horserace#6]
Currently if you have more than 25 or so columns, then there's no guarantee the final column have an axis tick, which means there's nothing to click to reveal the whole chart:

![image](https://user-images.githubusercontent.com/9587433/27131806-31f34e4e-5104-11e7-94eb-cca630244d8a.png)

We should give some proper thought to this. One idea would be to reintroduce a "play through" button which would always play from start to end.

**Comment by daanlouter** [comment]
I kept it simple for this version, only being able to hover/click on the ticks. In the future I think we should create an invisible tick for each data point on the x axis that is hoverable/clickable.

**Comment by theduncanclark** [comment]
Yes, that sounds sensible. I still think a play-through button might be nice, but let's see what potential users think.

**Comment by theduncanclark** [comment]
In the latest version which I'm about to push it plays through to the end by default, which makes this less urgent. But I'll leave the issue open as it's still a problem.
